public/polymer.coffee

```diff
@@ -61,7 +64,11 @@ sendMessage = (msg)->
     nick: template.userName
     avatar: template.avatar
 
+formatMessage = (msg)->
+  converter.makeHtml msg.replace urlRegex, '<a href="$1" target="_blank">$1</a>'
```

Comment:
use `simplifiedAutoLink` config option for showdown instead.

Reply:
That way we will loose `target="_blank"` functionality and I think it's important so that the user does not get moved away on clicking a link in the chat.

Reply:
regex and HTML is a minefield. I'd recommend rather running showdown with the autolink option, then do something like:

```js
var el = document.createElement('html');
el.innerHTML = body;
el.getElementsByTagName( 'a' ); // Live NodeList of your anchor elements
```

then iterate through all links and add a target=_blank.

Might also be worth pitching as an option to showdown perhaps?

Reply:
Turns out there is another option already there in showdown `openLinksInNewWindow`. Added that!